.github/workflows/publish.yml

```diff
@@ -18,5 +18,5 @@ jobs:
           export CONTAINER_LOWERCASE="${CONTAINER_IMAGE,,}"
           echo $CONTAINER_LOWERCASE
           docker login -u $GITHUB_ACTOR -p ${{secrets.GITHUB_TOKEN}} docker.pkg.github.com
-          docker build -t $CONTAINER_LOWERCASE .
+          docker build --build-arg PUBLIC_KEY=${{secrets.pubk}} --build-arg PRIVATE_KEY=${{secrets.privk}}-t $CONTAINER_LOWERCASE .
```

Comment:
Missing space before `-t`.
You also have to modify  Docker file save those args to file and then copy it with exisitng `COPY` commands